src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/helper/managehearings/HearingCorrespondenceHelper.java

```diff
@@ -61,6 +63,25 @@ public Hearing getHearingInContext(FinremCaseData finremCaseData) {
                 .getValue();
     }
 
+    public HearingTabItem getHearingInContextFromTab(FinremCaseData finremCaseData) {
+        ManageHearingsWrapper manageHearingsWrapper = finremCaseData.getManageHearingsWrapper();
+        UUID hearingId = manageHearingsWrapper.getWorkingHearingId();
+
+        List<HearingTabCollectionItem> hearingTabItems = manageHearingsWrapper.getHearingTabItems();
+
+        if (hearingTabItems == null) {
+            throw new IllegalStateException(
+                "No hearing tab items available to search for. Working hearing ID is: " + hearingId
+            );
+        }
+
+        return manageHearingsWrapper.getHearingTabItems().stream()
```

Comment:
use  `hearingTabItems`. And making use of `nullIfEmpty()` i.e. `nullIfEmpty(hearingTabItems)`